Bio/codonalign/codonalignment.py

```diff
@@ -126,22 +108,14 @@ def __add__(self, other):
                     "When adding two alignments they must have the same length"
                     " (i.e. same number or rows)"
                 )
-            if compare_codon_alphabet(self._alphabet, other._alphabet):
-                alpha = self._alphabet
-                # merged = (left + right for left, right in zip(self, other))
-                merged = (
-                    SeqRecord(
-                        seq=CodonSeq(
-                            str(left.seq) + str(right.seq), alphabet=left.seq.alphabet
-                        )
-                    )
-                    for left, right in zip(self, other)
-                )
-                return CodonAlignment(merged, alphabet=alpha)
-            else:
-                raise TypeError(
-                    "Only CodonAlignment with the same CodonAlphabet can be combined."
```

Comment:
I guess the situation is rare, but it is possible that the user may combine two codon alignments that don't share the same codon_table. Maybe we can have a warning here.

Reply:
Can you suggest a code snippet to add here?

Reply:
How about this?

`warnings.warn("Please make sure the two CodonAlignment objects are sharing the same codon table. This is not checked by Biopython.", BiopythonWarning)`

Reply:
I have updated the code, can you check to make sure is is what you meant?